task1/src/csv.js

```diff
@@ -0,0 +1,48 @@
+import { appendFileSync, existsSync, unlinkSync } from 'fs';
+import { csv } from 'csvtojson';
+
+const csvFilePath = './csv/input.csv';
+const txtFilePath = './txt/output.txt';
```

Comment:
As it was mentioned on our Q&A session it's recommended to use path.resolve

Reply:
the code was modified to use path.resolve